src/protocols/rdp/client.c

```diff
@@ -105,6 +105,7 @@ const char* GUAC_CLIENT_ARGS[] = {
     "enable-printing",
     "enable-drive",
     "drive-path",
+	"create-drive-path",
```

Comment:
Please indent with 4 spaces - no tabs. The style within the Guacamole source needs to be self-consistent.